pkgs/tools/networking/pwru/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, stdenv
+, buildGoModule
+, fetchFromGitHub
+, pkg-config
+, clang_13
+, llvmPackages_16
+, flex
+, bison
+, testers
+, pwru
+}:
+
+buildGoModule rec {
+  pname = "pwru";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "cilium";
+    repo = "pwru";
+    rev = "v1.0.0";
+    hash = "sha256-9UMuHDPDL8nPdL6U+BHHSXvd0iDUigVOa5Tel7n5L4Q=";
+  };
+
+  patches = [
+    ./go_generate_cflags.patch
+  ];
+
+  vendorHash = null;
+
+  CGO_ENABLED = 1;
```

Comment:
```suggestion
  env.CGO_ENABLED = "1";
```

Reply:
For my personal knowledge, why should it be in env? I don't find any derivation using this declaration.

Reply:
When I try your suggestion I also get this error: `error: The ‘env’ attribute set cannot contain any attributes passed to derivation. The following attributes are overlapping: CGO_ENABLED`. The buildGoModule has an explicit attribute for `CGO_ENABLED`: https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/go/module.nix#L35

Reply:
Sorry, I didn't noticed. I thought it was an environment variable, not a parameter.